packages/edge-config/src/index.ts

```diff
@@ -164,7 +164,12 @@ export function createClient(
 
   const baseUrl = connection.baseUrl;
   const version = connection.version; // version of the edge config read access api we talk to
-  const headers = { Authorization: `Bearer ${connection.token}` };
+  const headers: Record<string, string> = {
+    Authorization: `Bearer ${connection.token}`,
+  };
+
+  if (typeof process !== 'undefined' && process.env.NODE_ENV)
```

Comment:
It’s a bit weird that we’ll be sending NODE_ENV even from edge functions and middleware since it’s not node

Reply:
For consistency. I guess, ideally, the SDK wouldn't even know in which type of compute it runs.

Reply:
Since the refactoring which merged the entry points it isn’t as explicitly aware of it anymore as it was before. My comment was mainly about how we collectively ended up with having NODE_ENV in a runtime which isn’t actually node. No particular point, just found it a bit humorous

Reply:
Ohh, haven't even thought about that, true